### PR TITLE
Implement Home page

### PR DESCRIPTION
**Pull request by ana-maria-croitoru**
Create the home page of the app, which displays the list of restaurants

### CODE REVIEW

File: src/app/core/services/restaurant.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
```
Comment (AlexStoicaSM) [inline src/app/core/services/restaurant.service.spec.ts:2]: Remove empty space

File: src/app/features/restaurants/restaurants.component.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+export class RestaurantsComponent implements OnInit, OnDestroy {
+  restaurants: Restaurant[] = [];
+  restaurantsSubscription: Subscription;
+  constructor(private restaurantService: RestaurantService) {}
+
+  getRestaurants(): void {
+    this.restaurantsSubscription = this.restaurantService
+      .getRestaurants()
+      .subscribe((restaurants) => {
+        console.log(restaurants);
```
Comment (AlexStoicaSM) [inline src/app/features/restaurants/restaurants.component.ts]: Remove console.log

File: src/app/shared/card/card.component.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  border: 2px solid #e4e4e4;
+  background-color: #e4e4e4;
+}
+
+.card:hover {
+  background-color: #ffff;
+  border: 2px solid #ff1e23;
+}
+
+card-title {
```
Comment (AlexStoicaSM) [inline src/app/shared/card/card.component.scss]: Is this a class? you should add the "." in front of it

**Review by AlexStoicaSM — COMMENTED** [review]
Looks good, just one small change

File: src/app/shared/card/card.component.scss (hunk 3 above)
Comment (ana-maria-croitoru) [reply src/app/shared/card/card.component.scss]: Fixed

File: src/app/features/restaurants/restaurants.component.ts (hunk 2 above)
Comment (ana-maria-croitoru) [reply src/app/features/restaurants/restaurants.component.ts]: Fixed

**Review by AlexStoicaSM — APPROVED** [review]
LGTM

**Pull request merged**